easybuild/framework/easyblock.py

```diff
@@ -1444,8 +1443,19 @@ def extensions_step(self, fetch=False):
         self.clean_up_fake_module(fake_mod_data)
 
     def package_step(self):
-        """Package software (e.g. into an RPM)."""
-        pass
+        """Prepare package software (e.g. into an RPM) with fpm."""
+        rpmname = "HPCBIOS.20150211-%s-%s" % (self.name, self.version)
+        os.chdir(os.environ['TMPDIR'])
+        
+        if self.toolchain.name == "dummy":
+            dependencies = []
+        else:
+            dependencies = [ "=".join([ self.toolchain.name, self.toolchain.version ]) ]
+        dependencies.extend([ "=".join([ dep['name'], dep['version'] ]) for dep in self.cfg.dependencies() ])
+        depstring = '--dependency ' + ' --dependency '.join(dependencies)
+
+        cmd = "fpm --workdir $TMPDIR -t rpm --name %s -s dir %s -C %s" % (rpmname, depstring, self.installdir)
```

Comment:
Thanks, this is an interesting suggestion, as anything that removes troubles with fancy namings!